iac/terraform/cdk/appStack.ts

```diff
@@ -40,13 +40,13 @@ export class AppStack extends TerraformStack {
         super(scope, id)
         this.config = config
         console.log('config', config)
-
-        let arch = 'arm64'
-        const localArch = process.env.LOCAL_ARCH
-
-        if (config.isLocal && localArch == 'x86_64') {
-            arch = 'x86_64'
+        
```

Comment:
This is defaulting the architecture to undefined which is using the default `amd64` on my Mac M2 which isn't the intent. 
This is the code to run on `arm64` on my M2 and set to default to arm64 in AWS.
```
        let archList = ['arm64']
        const localArch = process.env.LOCAL_ARCH

        if (config.isLocal && localArch == 'x86_64') {
            archList = ['x86_64']
        }
```
Pic with your code 

<img width="755" alt="Screenshot 2023-12-20 at 11 11 45 AM" src="https://github.com/localstack-samples/multi-iac-devops/assets/850718/53f327e6-89d9-4591-9803-68082bcdd629">

Pic with code to set to native architecture on laptop and default to arm64 on AWS.
<img width="908" alt="Screenshot 2023-12-20 at 11 17 34 AM" src="https://github.com/localstack-samples/multi-iac-devops/assets/850718/9167aeb1-e56b-4d84-9183-3daefabb6e4e">

Reply:
Resolved now.